src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -162,7 +162,7 @@ def default_api_version(self):
             'role_definitions': '2018-01-01-preview',
             'provider_operations_metadata': '2018-01-01-preview'
         }),
-        ResourceType.MGMT_CONTAINERREGISTRY: '2019-12-01-preview',
+        ResourceType.MGMT_CONTAINERREGISTRY: '2020-11-01-preview',
```

Comment:
pls help to make sure run full live test for regression. :)

Reply:
I have re-run tests, please can you review @yungezz